packages/augur-ui/src/modules/modal/add-funds.tsx

```diff
@@ -58,42 +51,41 @@ const addFundsTorus = async (amount, address) => {
 
 export const AddFunds = ({
   autoSelect = false,
-  tokenToAdd = DAI,
 }: AddFundsProps) => {
+  const BUY_MIN = 20;
+  const BUY_MAX = 250;
   const { loginAccount, modal, actions: {closeModal} } = useAppStatusStore();
-  const isRelayDown = false;
-  const { useSigner, initialSwapToken, initialAddFundsFlow } = modal;
-  const showTransfer = modal.showTransfer || false;
+  const { env: config, ethToDaiRate, repToDaiRate } = useAppStatusStore();
 
-    //analyticsEvent: () => dP.track(ADD_FUNDS, {}),
   const closeAction = () => {
     if (modal.cb) {
       modal.cb();
     }
     closeModal();
   };
-    
 
-  const [amountToBuy, setAmountToBuy] = useState(createBigNumber(0));
-  const [isAmountValid, setIsAmountValid] = useState(false);
-  const [poolSelected, setPoolSelected] = useState(false);
-  const { env: config, ethToDaiRate, repToDaiRate } = useAppStatusStore();
   const ETH_RATE = createBigNumber(1).dividedBy(
```

Comment:
not sure why ETH_RATE/REP_RATE are defined here. just use the swap component below and don't pass props if we can just refer to the hook inside the leaf component (swap in this case)